### PR TITLE
CA2241: Added try_determine_additional_string_formatting_methods_auto…

### PR DESCRIPTION
**Pull request by manfred-brands**
The rule was recently updated with an additional configuration option.
https://github.com/dotnet/roslyn-analyzers/pull/3653
Updated the documentation to include this option.

### CODE REVIEW

**Comment by PRMerger9** [comment]
@manfred-brands : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by TerryGLee** [comment]
@Mikejo5000 Please take a look at this.

File: docs/code-quality/ca2241.md [hunk 1]
```diff
@@ -68,6 +67,13 @@ Examples:
 
 You can configure this option for just this rule, for all rules, or for all rules in this category (Usage). For more information, see [Configure FxCop analyzers](configure-fxcop-analyzers.md).
 
+Instead of a specifying a long list of additional string formatting methods, you can configure the analyzer to try and determine this automatically. By default this is disabled.
```
Comment (Mikejo5000) [inline docs/code-quality/ca2241.md:70]:
```suggestion
Instead of a specifying a long list of additional string formatting methods, you can configure the analyzer to try and determine this automatically. By default, this option is disabled.
```

File: docs/code-quality/ca2241.md [hunk 2]
```diff
@@ -68,6 +67,13 @@ Examples:
...
+If enabled then any method that has a 'string format' parameter followed by a 'params object[]' parameter is considered a string formatting method.
```
Comment (Mikejo5000) [inline docs/code-quality/ca2241.md:71]:
```suggestion
If enabled, then any method that has a `string format` parameter followed by a `params object[]` parameter is considered a string formatting method.
```

Comment (Mikejo5000) [reply docs/code-quality/ca2241.md:71]: inline code formatting for `string format` I believe. Is that correct?

File: docs/code-quality/ca2241.md (hunk 1 above)
Comment (mavasani) [inline docs/code-quality/ca2241.md:70]:
```suggestion
Instead of specifying an explicit list of additional string formatting methods, you can configure the analyzer to try and determine this automatically. By default, this option is disabled.
```

File: docs/code-quality/ca2241.md [hunk 3]
```diff
@@ -10,7 +10,6 @@ helpviewer_keywords:
 - ProvideCorrectArgumentsToFormattingMethods
 - CA2241
 ms.assetid: 83639bc4-4c91-4a07-a40e-dc5e49a84494
-author: mikejo5000
```
Comment (mavasani) [inline docs/code-quality/ca2241.md:13]: Why was this deleted? Should this change be reverted?

Comment (Mikejo5000) [reply docs/code-quality/ca2241.md:13]: Yes, not sure why that was deleted.

File: docs/code-quality/ca2241.md [hunk 4]
```diff
@@ -10,7 +10,6 @@ helpviewer_keywords:
 - ProvideCorrectArgumentsToFormattingMethods
 - CA2241
 ms.assetid: 83639bc4-4c91-4a07-a40e-dc5e49a84494
```
Comment (Mikejo5000) [inline docs/code-quality/ca2241.md:12]:
```suggestion
ms.assetid: 83639bc4-4c91-4a07-a40e-dc5e49a84494
author: mikejo5000
```

**Comment by PRMerger19** [comment]
@Mikejo5000 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger20** [comment]
@Mikejo5000 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/code-quality/ca2241.md [hunk 5]
```diff
@@ -68,6 +68,13 @@ Examples:
 
 You can configure this option for just this rule, for all rules, or for all rules in this category (Usage). For more information, see [Configure FxCop analyzers](configure-fxcop-analyzers.md).
 
+Instead of a specifying a long list of additional string formatting methods, you can configure the analyzer to try and determine this automatically. By default, this option is disabled.
```
Comment (Mikejo5000) [inline docs/code-quality/ca2241.md:71]:
```suggestion
Alternatively, instead of specifying an explicit list of additional string formatting methods, you can configure the analyzer to try and determine this automatically. By default, this option is disabled.
```

**Comment by PRMerger20** [comment]
@Mikejo5000 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/code-quality/ca2241.md (hunk 3 above)
Comment (Mikejo5000) [reply docs/code-quality/ca2241.md]: Added

File: docs/code-quality/ca2241.md (hunk 1 above)
Comment (Mikejo5000) [reply docs/code-quality/ca2241.md]: Merged this

**Comment by Mikejo5000** [comment]
#sign-off

**Comment by PRMerger15** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.
